docs/display/as-javascript-date.md

```diff
@@ -6,5 +6,5 @@ title: As Javascript Date
 To get a copy of the native `Date` object parsed from the Day.js object use `dayjs#toDate`.
 
 ```js
-dayjs('2019-01-25').toDate()
+dayjs('2019-01-25').toDate() // 'Fri Jan 25 2019 00:00:00 GMT-0500 (Eastern Standard Time)'
```

Comment:
it is better to add a comment to clear that this result is an JS object, and also on a specific timezone